mate-base/mate-settings-daemon/mate-settings-daemon-1.27.1.ebuild

```
@@ -0,0 +1,79 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MATE_LA_PUNT="yes"
+
+inherit mate
+
+MINOR=$(($(ver_cut 2) % 2))
+if [[ ${MINOR} -eq 0 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+fi
+
+DESCRIPTION="MATE Settings Daemon"
+LICENSE="GPL-2+ GPL-3+ HPND LGPL-2+ LGPL-2.1+"
+SLOT="0"
+
+IUSE="X accessibility debug libnotify policykit pulseaudio rfkill smartcard +sound"
+
+REQUIRED_USE="pulseaudio? ( sound )"
+
+COMMON_DEPEND=">=dev-libs/gdbus-codegen-2.76.4
```

```suggestion
COMMON_DEPEND=">=dev-util/gdbus-codegen-2.76.4
```

Since this ebuild isn't keyworded the CI doesn't catch this.

---

Fixed.